curvefs/proto/topology.proto

```diff
@@ -115,8 +115,8 @@ message Copyset {
 message MetaServerInfo {
     required uint32 metaServerID = 1;
     required string hostname = 2;
-    required string hostIp = 3;
-    required uint32 port = 4;
+    required string InternalIp = 3;
+    required uint32 InternalPort = 4;
```

Comment:
variable name should starts with lowercase

Reply:
> variable name should starts with lowercase

fixed